src/main/java/world/bentobox/challenges/database/object/requirements/StatisticRequirements.java

```diff
@@ -2,227 +2,139 @@
-            
-            case UNTYPED -> true;
-        
-        };
-
-    }
-
-
-    // ---------------------------------------------------------------------
-// Section: Getters and setters
-// ---------------------------------------------------------------------
-
+    @Expose
+    @Nullable
+    private Statistic statistic;
 
     /**
-     * Gets statistic.
-     *
-     * @return the statistic
+     * Type of entity for entity related statistics.
+     * @deprecated Shifting to a list
      */
+    @Expose
     @Nullable
-    public Statistic getStatistic()
-    {
-        return statistic;
-    }
-
+    private EntityType entity;
 
     /**
-     * Sets statistic.
-     *
-     * @param statistic the statistic
+     * Type of material for block and item related statistics.
+     * @deprecated Shifting to a list
      */
-    public void setStatistic(@Nullable Statistic statistic)
-    {
-        this.statistic = statistic;
-    }
-
+    @Expose
+    @Nullable
+    private Material material;
 
     /**
-     * Gets entity.
-     *
-     * @return the entity
+     * Amount of the stats.
+     * @deprecated Shifting to a list
      */
-    @Nullable
-    public EntityType getEntity()
-    {
-        return entity;
-    }
-
+    @Expose
+    private Integer amount;
 
     /**
-     * Sets entity.
-     *
-     * @param entity the entity
+     * Indicate that player statistic fields must be adjusted after completing challenges.
+     * @deprecated Shifting to a list
      */
-    public void setEntity(@Nullable EntityType entity)
-    {
-        this.entity = entity;
-    }
-
+    @Expose
+    private Boolean reduceStatistic;
 
     /**
-     * Gets material.
-     *
-     * @return the material
+     * List of statistics that must be done for this challenge
      */
+    @Expose
     @Nullable
-    public Material getMaterial()
-    {
-        return material;
-    }
+    private List<StatisticRec> statisticList;
 
 
     /**
-     * Sets material.
-     *
-     * @param material the material
+     * Constructor Requirements creates a new Requirements instance.
      */
-    public void setMaterial(@Nullable Material material)
+    public StatisticRequirements()
     {
-        this.material = material;
+        // Empty constructor
     }
 
 
     /**
-     * Gets amount.
-     *
-     * @return the amount
+     * This method copies given statistic object.
+     * @return Copy of this object.
      */
-    public int getAmount()
+    @Override
+    public Requirements copy()
     {
-        return amount;
+        StatisticRequirements requirements = new StatisticRequirements();
+        requirements.setStatisticList(this.getRequiredStatistics());
+        return requirements;
     }
 
 
-    /**
-     * Sets amount.
-     *
-     * @param amount the amount
-     */
-    public void setAmount(int amount)
+    @Override
+    public boolean isValid()
     {
-        this.amount = amount;
+        // TODO - do something here?
```

Comment:
@BONNe I'm not sure what this is for. It used to be some self-check code. Is it required still?